src/hooks/useDivisionOptions.tsx

```diff
@@ -1,7 +1,11 @@
-import { useNeighborhoodListQuery } from '../generated/graphql';
+import sortBy from 'lodash/sortBy';
+
+import { Neighborhood, useNeighborhoodListQuery } from '../generated/graphql';
 import getLocalisedString from '../util/getLocalisedString';
 import useLocale from './useLocale';
 
+const DIVISION_BLOCKLIST = ['kaupunginosa:aluemeri'];
```

Comment:
If there ever comes more of these, I would move it to some const-file.

Reply:
Hmmm maybe, but I think this is only going to be used in this helper hook and it only has only this one const. So is there  benefit by putting it to another file? Or do you mean like some global const file?